notebooks/practical_issues/PracticalIssues.ipynb

```diff
@@ -700,86 +700,86 @@
        "      <th>SVM (burden)</th>\n",
        "      <td>SVM</td>\n",
        "      <td>burden</td>\n",
-       "      <td>66.86</td>\n",
-       "      <td>63.74</td>\n",
-       "      <td>69.92</td>\n",
-       "      <td>66.86</td>\n",
-       "      <td>63.74</td>\n",
-       "      <td>69.92</td>\n",
-       "      <td>0.13</td>\n",
-       "      <td>0.12</td>\n",
-       "      <td>0.14</td>\n",
-       "      <td>0.10</td>\n",
-       "      <td>0.09</td>\n",
-       "      <td>0.12</td>\n",
-       "      <td>0.27</td>\n",
-       "      <td>0.25</td>\n",
-       "      <td>0.29</td>\n",
+       "      <td>61.624811</td>\n",
+       "      <td>58.660412</td>\n",
+       "      <td>64.771936</td>\n",
+       "      <td>61.624811</td>\n",
+       "      <td>58.660412</td>\n",
+       "      <td>64.771936</td>\n",
+       "      <td>0.135700</td>\n",
+       "      <td>0.122987</td>\n",
+       "      <td>0.148897</td>\n",
+       "      <td>0.107149</td>\n",
+       "      <td>0.097269</td>\n",
+       "      <td>0.117315</td>\n",
+       "      <td>0.323981</td>\n",
+       "      <td>0.301887</td>\n",
+       "      <td>0.348854</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>logistic (burden)</th>\n",
        "      <td>logistic</td>\n",
        "      <td>burden</td>\n",
-       "      <td>75.66</td>\n",
-       "      <td>72.94</td>\n",
-       "      <td>78.23</td>\n",
-       "      <td>75.66</td>\n",
-       "      <td>72.94</td>\n",
-       "      <td>78.23</td>\n",
-       "      <td>0.15</td>\n",
-       "      <td>0.14</td>\n",
-       "      <td>0.18</td>\n",
-       "      <td>0.12</td>\n",
-       "      <td>0.11</td>\n",
-       "      <td>0.15</td>\n",
-       "      <td>0.25</td>\n",
-       "      <td>0.24</td>\n",
-       "      <td>0.29</td>\n",
+       "      <td>58.705817</td>\n",
+       "      <td>55.863313</td>\n",
+       "      <td>61.634591</td>\n",
+       "      <td>58.705817</td>\n",
+       "      <td>55.863313</td>\n",
+       "      <td>61.634591</td>\n",
+       "      <td>0.116009</td>\n",
+       "      <td>0.104068</td>\n",
+       "      <td>0.134025</td>\n",
+       "      <td>0.089267</td>\n",
+       "      <td>0.080139</td>\n",
+       "      <td>0.102750</td>\n",
+       "      <td>0.294707</td>\n",
+       "      <td>0.272967</td>\n",
+       "      <td>0.335061</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
        "                    context    type  overall fairness  \\\n",
-       "SVM (burden)            SVM  burden             66.86   \n",
```

Comment:
I'm seeing big changes in the scores here (and below). Is this expected?

Reply:
Only in the sense that I didn't touch this notebook at all (deliberately), and did not consciously rerun it as part of this work!  In fact this PR should just be the addition of one new notebook.  I must have fat-fingered by commit somehow.  I will fix it and force push the correct version (just the new notebook added).  No idea what vintage these accidentally checked in notebooks are form, but they are spurious